GoogleDataTransport/GDTCCTLibrary/GDTCCTUploader2.m

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "GoogleDataTransport/GDTCCTLibrary/Private/GDTCCTUploader.h"
+
+#import "GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/GDTCORConsoleLogger.h"
+#import "GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/GDTCOREvent.h"
+#import "GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/GDTCORPlatform.h"
+#import "GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/GDTCORRegistrar.h"
+
+#import "GoogleDataTransport/GDTCCTLibrary/Private/GDTCCTUploadOperation.h"
+
+@interface GDTCCTUploader () <NSURLSessionDelegate>
```

Comment:
The file name here is GDTCCUploader2.m. Is this intended? I'm expecting this to be GDTCCTUploader.m

Reply:
Renaming `GDTCCUploader.m` is a trick that allows to make the diff more readable. When old version of  `GDTCCUploader.m` is removed then git diff rename detection picks `GDTCCTUploadOperation.m` as a rename and presents diff only. Otherwise `GDTCCTUploadOperation.m` would look like a completely new file which would create  extra noise. 

In such cases I'm missing google3 where you can explicitly branch a file :(